requre/cassette.py

```diff
@@ -21,14 +21,670 @@
 # SOFTWARE.
 from pathlib import Path
 
-from requre.storage import PersistentObjectStorage
+import logging
+from typing import Dict, Optional, List, Hashable, Any, Callable
+from enum import Enum
+import os
+import yaml
+import time
+import inspect
 
+from requre.utils import StorageMode
+from requre.constants import (
+    METATADA_KEY,
+    ENV_REQURE_STORAGE_MODE,
+    ENV_STORAGE_FILE,
+    VERSION_REQURE_FILE,
+    KEY_MINIMAL_MATCH,
+)
+from requre.exceptions import (
+    PersistentStorageException,
+    ItemNotInStorage,
+    StorageNoResponseLeft,
+)
+
+# use this sleep to avoid decorating original time function used internally
+original_sleep = time.sleep
+original_time = time.time
+logger = logging.getLogger(__name__)
+
+
+class StorageKeysInspect:
```

Comment:
What about having this somewhere else?

- And have only cassette class here.
- What about making `online_replacing` a module?

Reply:
We agreed that we will create a good-firs-issues for that.